src/js/components/introduction.jsx

```diff
@@ -26,35 +25,30 @@ export class IntroductionComponent extends Component {
 
     constructor(...args) {
         super(...args);
-        this._debounceHeightCalculation = debounce(this.calculateIntroHeight.bind(this), 150);
+        this._debounceClientHeightCalculation = debounce(this.calculateIntroHeight.bind(this), 150);
     }
 
     componentDidMount() {
-        setTimeout(() => this.calculateIntroHeight());
-        window.addEventListener('resize', this._debounceHeightCalculation);
-    }
+        // Height of Introduction component will be computed on render and on resize only
+        // Make sure Introduction component has fully rendered before computing height
+        setTimeout(() => {
```

Comment:
is there any way to synchronize with intro rendering completion that doesn't rely on a setTimeout like this? Could we synchronize on `componentDidUpdate` for example?